internal/operators/ocs/ocs_operator.go

```diff
@@ -95,3 +96,25 @@ func (o *operator) GetMonitoredOperator() *models.MonitoredOperator {
 func (o *operator) GetHostRequirements(context.Context, *common.Cluster, *models.Host) (*models.ClusterHostRequirementsDetails, error) {
 	return &models.ClusterHostRequirementsDetails{}, nil
 }
+
+// GetPreflightRequirements returns operator hardware requirements that can be determined with cluster data only
+func (o *operator) GetPreflightRequirements(context.Context, *common.Cluster) (*models.OperatorHardwareRequirements, error) {
+	return &models.OperatorHardwareRequirements{
+		OperatorName: o.GetName(),
+		Dependencies: o.GetDependencies(),
+		Requirements: &models.HostTypeHardwareRequirementsWrapper{
+			Master: &models.HostTypeHardwareRequirements{
+				// TODO: adjust when https://github.com/openshift/assisted-service/pull/1456 is merged
+				Quantitative: &models.ClusterHostRequirementsDetails{},
```

Comment:
OCS has the same qualitative requirements in case of an all master cluster (compact  mode). Should that be added here as well?
OCS also requires a minimum of 3 nodes.